Neo4j.Driver/common.props

```diff
@@ -0,0 +1,6 @@
+<Project>
+	<PropertyGroup>
+		<XunitVersion>2.3.0-beta2-*</XunitVersion>
```

Comment:
Do we need to use beta version? Seems like they are approaching GA soon. https://www.nuget.org/packages/xunit/

Reply:
In order to use the new cross-platform testing infrastructure, this is the package we've to use. However, since this is an improvement we can wait for its release and leave this PR open.